specification/security_model.adoc

```diff
@@ -1,9 +1,195 @@
+  access to the system and its devices, and can use simple or advanced hardware
+  attacks such as bus interposers, voltage/clock glitchers, microscopes, power
+  usage analyzer, etc. in order to access or tamper with the protected assets.
+
+=== Threats
+
+==== CoVE-IO-T001
+
+.CoVE-IO-T001
+|===
+| Asset | Threat | Adversary | Scope | Result
+
+| TVM confidential data
+| Disclosure
+| Privileged host software
+| In scope
+| Host component reads TVM confidential data
+
+5+| **Description** - A privileged host software component programs a TDI that
+    is assigned to a TVM. By accessing the device MMIO space, the host component
+    can program direct memory access destination addresses to its own address
+    space. Device generated data intended to be copied to the TVM is accessed by
+    the host component instead.
+
+|===
+
+==== CoVE-IO-T002
+
+.CoVE-IO-T002
+|===
+| Asset | Threat | Adversary | Scope | Result
+
+| TVM confidential data
+| Tamper
+| Privileged host software
+| In scope
+| Host component writes to TVM confidential memory
+
+5+| **Description** - A privileged host software component programs a TDI that
+    is assigned to a TVM. By accessing the device MMIO space, the host component
+    can modify the programmed direct memory access destination addresses, and
+    redirect it to unintended parts of the TVM address space.
+
+|===
+
+==== CoVE-IO-T003
+
+.CoVE-IO-T003
+|===
+| Asset | Threat | Adversary | Scope | Result
+
+| TVM confidential data
+| Denial of service
+| Privileged host software
+| **Not** in scope
+| TVM can not access a TDI that is assigned to it
+
+5+| **Description** - A privileged host software component resets or powers down
+    an assigned TDI or its physical device, while the TDI is assigned to a TVM.
+    The TVM is no longer able to directly access its assigned TDI.
+
+|===
+
+==== CoVE-IO-T004
+
+.CoVE-IO-T004
+|===
+| Asset | Threat | Adversary | Scope | Result
+
+| System memory
+| Tamper
+| Privileged guest software
+| In scope
+| TVM writes to system memory
+
+5+| **Description** - A privileged guest software component programs its
+    assigned TDI direct memory access destination addresses to be within
+    a region of the system address space that it is not allowed to access.
+    The host memory is tampered with the TDI generated data.
+
+|===
+
+==== CoVE-IO-T005
+
+.CoVE-IO-T005
+|===
+| Asset | Threat | Adversary | Scope | Result
+
+| System memory
+| Tamper
+| Privileged guest software
+| In scope
+| TVM writes to system memory
+
+5+| **Description** - A privileged guest software component programs its
+    assigned TDI direct memory access destination addresses to be within
+    a region of the system address space that it is not allowed to access.
+    The host memory is tampered with the TDI generated data.
+
+|===
+
+==== CoVE-IO-T006
+
+.CoVE-IO-T006
+|===
+| Asset | Threat | Adversary | Scope | Result
+
+| Device MMIO
```

Comment:
It should be "device trusted-MMIO", right?

A shared-MMIO does not have security properties.